pwa/src/hooks/useAuthentication.tsx

```diff
@@ -102,3 +102,24 @@ export const validateSession = () => {
 
   return !expired;
 };
+
+export const checkIfExpiresSoon = () => {
```

Comment:
We should add a return type (e.g. `:boolean`). And maybe we can rename this to `shouldRenewToken`, instead of `checkIfExpiresSoon`.